src/widgets/smart-home/entity-state/entity-state.widget.tsx

```diff
@@ -0,0 +1,94 @@
+import { Center, Loader, Stack, Text, Tooltip } from '@mantine/core';
+import { IconAlertHexagon, IconBraces, IconExclamationMark } from '@tabler/icons-react';
+import { useTranslation } from 'react-i18next';
+import { useConfigContext } from '~/config/provider';
+import { api } from '~/utils/api';
+import { defineWidget } from '~/widgets/helper';
+import { IWidget } from '~/widgets/widgets';
+
+const definition = defineWidget({
+  id: 'smart-home/entity-state',
+  icon: IconBraces,
+  options: {
+    entityId: {
+      type: 'text',
+      defaultValue: 'sun.sun',
+      info: true
+    },
+    displayName: {
+      type: 'text',
+      defaultValue: 'Sun'
+    },
+  },
+  gridstack: {
+    minWidth: 1,
```

Comment:
Are you sure the widget displays correctly for a 1x1 size ?

Reply:
Yes